miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -1687,14 +1685,18 @@ public static PoolDto asDto(@Nonnull Pool from, boolean includeContents, boolean
       }
       dto.setPooledElements(pooledElements);
 
-      dto.setDuplicateIndicesSequences(from.getDuplicateIndicesSequences());
+      dto.setDuplicateIndicesSequences(indexChecker.getDuplicateIndicesSequences(from));
       dto.setDuplicateIndices(dto.getDuplicateIndicesSequences() != null && !dto.getDuplicateIndicesSequences().isEmpty());
-      dto.setNearDuplicateIndicesSequences(from.getNearDuplicateIndicesSequences());
+      dto.setNearDuplicateIndicesSequences(indexChecker.getNearDuplicateIndicesSequences(from));
```

Comment:
Missing null checks on `indexChecker` here and above

Reply:
Added